scss/_patterns_muted-heading.scss

```
@@ -2,5 +2,11 @@
 @mixin vf-p-muted-heading {
   .p-muted-heading {
     @extend %small-caps-text;
+    
+    margin-bottom: #{map-get($sp-after, p) - .5rem - map-get($nudges, p)};
```

@lyubomir-popov I think you can use calc here, so SASS can be smart enough to calculate what is possible on build time.

```suggestion
    margin-bottom: calc(map-get($sp-after, p) - .5rem - map-get($nudges, p));
```

But also, why is this so complicated? Weren't nudges supposed to be part of top padding to keep things on baseline grid? Do we use them for bottom margins as well? If so, why aren't they just part of `map-get($sp-after, p)`? If we have specific variable for spacing after paragraph, why do we need to remove nudge from it?

With calculation like this is practically impossible to maintain this. No one in future will know what is the expected margin supposed to be and how to modify it. If we need any calculation here, it needs to be expressed in comment what is it calculating.

When we introduced `p-text--small-caps` it was supposed to be replacing p-muted-heading, now muted heading is using same text style as small caps text, but has different margin. Is this expected? Will designers / devs know which one is which?